src/oauth-handler.js

```diff
@@ -31,7 +32,9 @@ export default function oauth(Client, {
   shipCache = null
 }) {
   function getURL(req, url, qs = { token: req.hull.token }) {
-    return `https://${req.hostname}${req.baseUrl}${url}?${querystring.stringify(qs)}`;
+    const host = `https://${req.hostname}${req.baseUrl}${url}`;
+    if (qs === false) return host;
```

Comment:
the reason I wanted to have a choice is that in some other place i append "&reset=true" to the url. Not having "?" makes the url invalid. I wanted to have a choice there.